### PR TITLE
upgrade to iota_stronghold 2.0.0

### PR DESCRIPTION
**Pull request by andrewweston**
As discussed with @FabianLars, I have upgraded `iota_stronghold` to v2.0.0. The plugin compiles (the sql plugin is preventing a clean build) and works in my test app

### CODE REVIEW

File: plugins/stronghold/Cargo.toml [hunk 1]
```diff
@@ -21,11 +21,11 @@ serde_json = { workspace = true }
 tauri = { workspace = true }
 log = { workspace = true }
 thiserror = { workspace = true }
-iota_stronghold = "1"
+iota_stronghold = "2.0.0"
```
Comment (FabianLars) [inline plugins/stronghold/Cargo.toml]:
```suggestion
iota_stronghold = "2"
```
just to make renovate less annoying lol

File: plugins/stronghold/Cargo.toml [hunk 2]
```diff
@@ -21,11 +21,11 @@ serde_json = { workspace = true }
...
 iota-crypto = "0.23"
 hex = "0.4"
 zeroize = { version = "1", features = [ "zeroize_derive" ] }
-rust-argon2 = { version = "1", optional = true }
+rust-argon2 = { version = "2.1.0", optional = true }
```
Comment (FabianLars) [inline plugins/stronghold/Cargo.toml]:
```suggestion
rust-argon2 = { version = "2", optional = true }
```
same

**Review by FabianLars — CHANGES_REQUESTED** [review]
awesome, thank you :)

**Comment by andrewweston** [comment]
Apologies for the delay in getting back to this, I had a busier week than I was expecting. It looks like you may have already made the changes?

**Review by FabianLars — APPROVED** [review]

**Pull request closed without merging**